tests/pythonMeep.bash

```diff
@@ -0,0 +1,8 @@
+#/bin/bash
+
+#
+# test for successful loading of python meep module
+#
+cd ../python
+ln -f -s .libs/_meep.so
+python -c 'import meep'
```

Comment:
Regarding` -gslcblas`, I was under the impression that this is independent of the low-level BLAS provided by packages such as openblas or atlas. The routines in `gslcblas` have `cblas_` prefixes that aren't present in the routines provided by openblas. All of my codes compile with optional GSL support, and linking with both `-lgslcblas` and `-lopenblas` doesn't seem to cause problems. Am I missing something on this?